[2025-08-02T13:21:23.288+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: marine_data_etl_dag.sst_etl.load_sst manual__2025-08-02T13:19:36+00:00 [queued]>
[2025-08-02T13:21:23.306+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: marine_data_etl_dag.sst_etl.load_sst manual__2025-08-02T13:19:36+00:00 [queued]>
[2025-08-02T13:21:23.307+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-08-02T13:21:23.332+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): sst_etl.load_sst> on 2025-08-02 13:19:36+00:00
[2025-08-02T13:21:23.339+0000] {standard_task_runner.py:60} INFO - Started process 336 to run task
[2025-08-02T13:21:23.346+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'marine_data_etl_dag', 'sst_etl.load_sst', 'manual__2025-08-02T13:19:36+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/marine_data_etl_dag.py', '--cfg-path', '/tmp/tmprk99ox38']
[2025-08-02T13:21:23.351+0000] {standard_task_runner.py:88} INFO - Job 89: Subtask sst_etl.load_sst
[2025-08-02T13:21:23.433+0000] {task_command.py:423} INFO - Running <TaskInstance: marine_data_etl_dag.sst_etl.load_sst manual__2025-08-02T13:19:36+00:00 [running]> on host 19821aecac76
[2025-08-02T13:21:23.786+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='marine_data_etl_dag' AIRFLOW_CTX_TASK_ID='sst_etl.load_sst' AIRFLOW_CTX_EXECUTION_DATE='2025-08-02T13:19:36+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-02T13:19:36+00:00'
[2025-08-02T13:21:23.848+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-02T13:21:23.873+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=marine_data_etl_dag, task_id=sst_etl.load_sst, execution_date=20250802T131936, start_date=20250802T132123, end_date=20250802T132123
[2025-08-02T13:21:23.920+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-02T13:21:23.979+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
